lambda/custom/document/renderPage.json

```diff
@@ -47,7 +47,7 @@
             "type" : "Frame" , 
             "item" : { 
                 "type" : "Image" , 
-                "source" : "",
+                "source" : "${payload.phrase.properties.logoUrl}",
```

Comment:
Why is the logo url underneath the phrase object? Can you make this payload.properties.logoUrl instead?

Reply:
done